### PR TITLE
feature/custom_marker

### PR DESCRIPTION
**Pull request by pgilliar**
add the posilibty to chose custom icon for marker

### CODE REVIEW

File: app/src/main/java/com/github/palFinderTeam/palfinder/meetups/MeetUp.kt [hunk 1]
```diff
@@ -156,6 +159,7 @@ data class MeetUp(
         const val HAS_CAPACITY = "hasMaxCapacity"
         const val CAPACITY = "capacity"
         const val ICON = "icon"
+        const val MARKER = "marker"
```
Comment (TruddyTheDuddi) [inline app/src/main/java/com/github/palFinderTeam/palfinder/meetups/MeetUp.kt:162]: What is exactly this ID? Is it the ID of an image in the library of these marker icons, or is it the ID number of the picture inside the android folders?

**Review by TruddyTheDuddi — APPROVED** [review]
Nice work! We will need to figure out how to implement it in the design of the app, as I suppose for now the modifications to the .xml of the MeetupCreation are temporary. Also we can see how to display these on the Map.

File: app/src/main/java/com/github/palFinderTeam/palfinder/meetups/MeetUp.kt (hunk 1 above)
Comment (pgilliar) [reply app/src/main/java/com/github/palFinderTeam/palfinder/meetups/MeetUp.kt:162]: It's the id of the image in a local icon pack

**Review by zaclegarssure — APPROVED** [review]
Looks fine but we should be careful with the coverage.

**Pull request merged**